run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol

```diff
@@ -1465,10 +1508,17 @@ Warnings:                    1
 0 CORBETT_SHIP_SO2     $ROOT/CORBETT_SHIP/v2014-07/CORBETT_ship.geos.1x1.nc            SO2_SHIP        1985/1-12/1/0          C xy kg/m2/s  SO2  -           10 3
 )))CORBETT_SHIP
 
-(((HTAP_SHIP
-0 HTAP_SHIP_SO2        $ROOT/HTAP/v2015-03/EDGAR_HTAP_SO2_SHIPS.generic.01x01.nc       emi_so2         2008-2010/1/1/0        C xy kg/m2/s  SO2  11/29       10 4
-0 HTAP_SHIP_SOAP       $ROOT/HTAP/v2015-03/EDGAR_HTAP_CO_SHIPS.generic.01x01.nc        emi_co          2008-2010/1/1/0        C xy kg/m2/s  SOAP 6/28/280    10 4
-)))HTAP_SHIP
+(((HTAPv3_SHIP
+0 HTAPv3_CO_SHP   $ROOT/HTAPv3/v2022-12/$YYYY/HTAPv3_CO_0.1x0.1_$YYYY.nc  CO_SHP  2000-2018/1-12/1/0 C xy kg/m2/s SOAP  26/280 10 4
```

Comment:
This field should be renamed to `HTAPv3_SOAP_SHP`

Reply:
Now fixed in 3f69125dbdf6ae180be14d943941683f4c7b9185
https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.aerosol#L1511-L1512